public/index.pug

```diff
@@ -29,22 +29,40 @@ html(lang="en-US")
         li(class="header__list-item")
-          a(href="https://unpkg.com/browse/simple-icons/") Unpkg
+          a(class="header__list-icon" title="Unpkg (CDN)" href="https://unpkg.com/browse/simple-icons/" rel="noopener") 
+            svg(role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg")
+              path(d="M 12.000002,-1.0439e-4 A 12.000002,12.000002 0 0 0 0,11.999898 12.000002,12.000002 0 0 0 12.000002,23.9999 12.000002,12.000002 0 0 0 24.000005,11.999898 12.000002,12.000002 0 0 0 12.000002,-1.0439e-4 Z m 0,2.00178449 A 10.008923,10.008923 0 0 1 22.008925,12.010603 10.008923,10.008923 0 0 1 12.000002,22.019525 10.008923,10.008923 0 0 1 1.9910798,12.010603 10.008923,10.008923 0 0 1 12.000002,2.0016801 Z m 2.140946,3.9286359 -0.03126,4.046388 -0.03329,4.046388 c -0.730063,2.108832 -3.414809,2.108832 -3.992864,0.007 C 10.008601,13.719677 9.9657825,12.23172 9.9657825,9.7482222 V 5.9373383 H 6.8400013 l 0.04314,4.1427307 c 0.053417,4.474577 0.08446,4.742195 0.6615523,5.909011 0.3532561,0.685103 1.2096345,1.445138 1.9696703,1.723461 1.3916151,0.524532 3.6824271,0.524532 5.0740421,0 0.749331,-0.278323 1.552186,-0.975201 1.937556,-1.680642 0.58876,-1.102587 0.654059,-1.627119 0.654059,-6.016059 l 0.0098,-4.0142733 -3.040143,-0.064657 z")
         li(class="header__list-item")
-          a(href="https://github.com/simple-icons/simple-icons/blob/develop/DISCLAIMER.md") Disclaimer
+          a(class="header__list-icon" title="Open Collective" href="https://opencollective.com/simple-icons" rel="noopener") 
+            svg(role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg")
+              path(d="M12 0C5.373 0 0 5.373 0 12s5.373 12 12 12c2.54 0 4.894-.79 6.834-2.135l-3.107-3.109a7.715 7.715 0 1 1 0-13.512l3.107-3.109A11.943 11.943 0 0 0 12 0zm9.865 5.166l-3.109 3.107A7.67 7.67 0 0 1 19.715 12a7.682 7.682 0 0 1-.959 3.727l3.109 3.107A11.943 11.943 0 0 0 24 12c0-2.54-.79-4.894-2.135-6.834z")
+        li(class="header__list-item")
+          a(class="header__list-icon" title="Legal disclaimer" href="https://github.com/simple-icons/simple-icons/blob/master/README.md" rel="noopener") 
+            svg(role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" stroke="#fff")
```

Comment:
I think the `stroke` should be removed here, it [looks a little bit weird in dark mode](https://user-images.githubusercontent.com/15719479/130332670-c5f7b10d-713e-4380-b10d-8ee9ac7d97ed.png)

```suggestion
            svg(role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg")
```

Reply:
uff, yes. what an oversight on my part.